railties/lib/rails/generators/erb/scaffold/templates/new.html.erb.tt

```diff
@@ -1,5 +1,9 @@
-<h1>New <%= singular_table_name.titleize %></h1>
+<h1>New <%= singular_table_name %></h1>
```

Comment:
Why was titleize removed from here? Do we want to generate `New user_account`?

Reply:
Ahh, I was just looking to get rid of the capitalization. I guess we'd need to do #human_name instead?